pkg/lib/registry/registry.go

```diff
@@ -523,3 +445,75 @@ func isDeprecated(ctx context.Context, q *sqlite.SQLQuerier, bundle registry.Bun
 	}
 	return false, nil
 }
+
+/* opm index add drops bundles that replace a channel head, and since channel head selection heuristics
```

Comment:
nit: go doc style asks that long function descriptions are put beneath the short function description.

(not super important because private methods aren't included in go docs anyway)